README.md

```diff
@@ -1,35 +1,53 @@
-# ExercismWinSetup
+# Windows Installer for Exercism Windows CLI
 A project to download and install the latest version of Exercism Client for Windows depending on the architecture.
 
 Exercism is licensed under MIT License. Copyright (c) 2013 Mike Gehard, Katrina Owen
```

Comment:
This should be removed. There's a LICENSE file.

Also, if possible, I'd like to assign copyright to `Exercism, Inc` (I've done so in all the track repositories). I created the entity about 7 months ago so that I can start looking for sponsorship, and for some simple legal protection.